plugins/meson.build

```diff
@@ -21,6 +21,7 @@ plugins = {
   'acpi-ivrs': false,
   'acpi-phat': false,
   'algoltek-usb': false,
+  'algoltek-aux': false,
```

Comment:
for alphabetica order, this should come before `algoltek-usb`

Reply:
OK! Already edited.